src/source/Utils/ExponentialBackoffUtils.c

```diff
@@ -0,0 +1,221 @@
+#include "ExponentialBackoffUtils.h"
+#include "../Include_i.h"
+
+STATUS getDefaultExponentialBackOffConfig(PExponentialBackoffConfig* ppPExponentialBackoffConfig) {
```

Comment:
Suggest naming the API more of createDefaultExponentialBackoffConfig. The pattern used is more of create/free

Reply:
Since we're creating state struct using default config values, I renamed it to **createExponentialBackoffStateWithDefaultConfig**